src/pages/dashboard/_UsAreaChartContainer.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { StaticQuery, graphql } from 'gatsby'
+import AreaChart from './charts/_AreaChart'
+import { parseDate } from './_util'
+
+export default function UsAreaChartContainer() {
+  const transformData = data => {
+    const transformedData = data.allCovidUsDaily.edges
+      .map(({ node }) => [
+        {
+          date: parseDate(node.date),
+          label: 'Total',
+          value: node.totalTestResults,
+        },
+        { date: parseDate(node.date), label: 'Positive', value: node.positive },
+      ])
+      .flat()
+    return transformedData
+  }
+  const query = graphql`
```

Comment:
This could also be shorter with [useStaticQuery](https://www.gatsbyjs.org/docs/use-static-query/#basic-example)

```javascript
const data = useStaticQuery(graphql`... your query ...`)

return (
<AreaChart data={data} ..../>
)
```

Reply:
Ooh nice, I love hooks, I'll switching over to using this.